### PR TITLE
Congress: Adjust look based on the new design

### PR DESCRIPTION
**Pull request by jagtalon**
### This is the proposed design:

mobile - representatives 5

### Here's how it currently looks:

screen shot 2016-01-18 at 9 19 58 am

screen shot 2016-01-18 at 9 31 05 am

Currently a WIP because I still have to test it extensively, and sync up with @chrismorast to see if the changes are sufficient.

---

https://duck.co/ia/view/congress

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by jagtalon** [comment]
screen shot 2016-01-18 at 10 48 51 am
screen shot 2016-01-18 at 10 49 04 am
screen shot 2016-01-18 at 10 49 39 am
screen shot 2016-01-18 at 10 49 48 am

**Comment by jagtalon** [comment]
Ready for review!

**Comment by jagtalon** [comment]
@chrismorast has already reviewed the design side:

screen shot 2016-01-18 at 12 36 47 pm

File: share/spice/congress/congress.css [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
 }
 
+/* The images are a bit weird, so it's best to adjust the height of the 
+   tile to see more of the image. */
 .tile--congress .tile__media {
-    margin-top: 1.4em;
-    height: 110px;
-    width: 110px;
-    border-radius: 100%;
+    height: 40%;
```
Comment (moollaza) [inline share/spice/congress/congress.css]: We typically set the height in `em`'s

File: share/spice/congress/footer.handlebars [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{{#if district}}
+    <span class="tile__sub">{{district}} District</span>
```
Comment (moollaza) [inline share/spice/congress/footer.handlebars]: if you use `<div>` with class `tx-clr--grey` you can ditch the `tile__sub` and the extra CSS to adjust the margin

**Comment by moollaza** [comment]
@jagtalon This LGTM :+1:

**Pull request merged**